.gitignore

```diff
@@ -1,2 +1,18 @@
 *~
 .DS_Store
+
+examples/react-todos/packages/npm-container/.npm/package/npm-shrinkwrap.json
```

Comment:
I can't see how to undo this change in the GitHub for Mac client. It shows all my commit's except this. re: `.gitignore`